docs/hardhat-network/README.md

```diff
@@ -371,6 +387,7 @@ To customise it, take a look at [the configuration section](/config/README.md#ha
 - `evm_revert` – same as Ganache.
 - `evm_snapshot` – same as Ganache.
 - `evm_setNextBlockTimestamp` - this method works like `evm_increaseTime`, but takes the exact timestamp that you want in the next block, and increases the time accordingly.
+- `evm_setMinGasPrice` - change the minimum gas price accepted by the network (in wei)
```

Comment:
I think this should be `hardhat_`

Reply:
Done in 8cd2d41d9cd8e0259e83ec8033b59e9548bf83a6.